softdevice_controller/CHANGELOG.rst

```diff
@@ -46,14 +46,17 @@ Bugfixes
 Known issues and limitations
 ============================
 
-* The RSSI value reported by the Softdevice Controller is the raw value from the radio peripheral.
-  Some SoCs require compensation of the RSSI value based on the chip temperature.
-  See the Errata document for the respective SoC for detailed information.
+The RSSI value reported by the Softdevice Controller is the raw value from the radio peripheral.
+Some SoCs require compensation of the RSSI value based on the chip temperature.
+See the Errata document for the respective SoC for detailed information.
```

Comment:
On second thought, I think this limitation should go into known issues in sdk-nrf in some form as well. Some of the known issues listed there will never be fixed, so they are technically speaking limitations.

We could still keep this limitation here or maybe move it to the API doc for the function that reports RSSI?

Reply:
Would like to hear from you, @ru-fu and @OisinFitz .

Reply:
If you want to move it to Known issues, move it. I don't think it's a good idea to add issues that will never be fixed to that page though, and if we start doing this, the page will just turn into a catch-all for everything that we don't know where to put. But your call.

And as I explained in detail in our last meeting, there is no API doc where we could put this limitation.